pkg/network/dhcpconfigurator/dhcp_configurator.go

```diff
@@ -29,3 +54,26 @@ func (d DhcpConfigurator) ImportConfiguration(ifaceName string) (*cache.DhcpConf
 func (d DhcpConfigurator) ExportConfiguration(config cache.DhcpConfig) error {
 	return d.dhcpConfigurationFactory.CacheDhcpConfigForPid(d.launcherPID).Write(config.Name, &config)
 }
+
+func (d DhcpConfigurator) EnsureDhcp(podInterfaceName string, dhcpConfig cache.DhcpConfig, dhcpOptions *v1.DHCPOptions) error {
```

Comment:
Maybe already wrote it somewhere else, I would all the methd "StartServerIfNeeded`.